terraform/organizations-policies.tf

```diff
@@ -0,0 +1,57 @@
+# Note that every AWS account in an AWS organisation _must_ have at least one SCP set.
+
+# This policy is the default set by AWS for every account in an AWS organisation.
+# There is a default policy titled "FullAWSAccess" that you can't edit.
+# See: https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples.html
+
+# Alexa business deny
+resource "aws_organizations_policy" "alexa-business-deny" {
+  name        = "Alexa business deny"
+  description = "Alexa business deny"
```

Comment:
This doesn't feel like it tells me what this actually is?

Reply:
Indeed, it does not - I just wanted to take it as-is from the console before I clean it up.